stacks/spica/src/function/pages/add/add.component.ts

```diff
@@ -24,8 +24,13 @@ import {
   emptyTrigger,
   Information,
   NormalizedFunction,
-  normalizeFunction
+  normalizeFunction,
+  Trigger
 } from "../../interface";
+import {MatDialog} from "@angular/material/dialog";
+import {CodeComponent} from "./code/code.component";
+import * as ts from "prettier/parser-typescript";
+import {format} from "prettier/standalone";
```

Comment:
Importing these modules within the main bundle causes a significant bundle size growth. So we can not do something like this. 

I think the way to go is by doing something we'd done in https://github.com/spica-engine/spica/blob/master/stacks/spica/src/function/components/editor/snippets.ts.